api/db/migrations/20230725115718_add_created_by_and_detached_at_in_complementary_certification_badges.js

```diff
@@ -0,0 +1,19 @@
+const TABLE_NAME = 'complementary-certification-badges';
+const CREATED_BY_COLUMN = 'createdBy';
+const DETACHED_AT_COLUMN = 'detachedAt';
+
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, async (table) => {
+    table.dateTime(DETACHED_AT_COLUMN).nullable();
+    table.integer(CREATED_BY_COLUMN).references('users.id');
```

Comment:
Commentaire après le merge 

integer simple est limitant ici
```suggestion
    table.bigInteger('userId').unsigned().references('users.id');
```